scipy/integrate/odepack.py

```diff
@@ -28,6 +28,9 @@ def odeint(func, y0, t, args=(), Dfun=None, col_deriv=0, full_output=0,
            mxords=5, printmessg=0):
     """
     Integrate a system of ordinary differential equations.
+    
+    .. note:: Use of this function is not recommended. Instead, use
```

Comment:
That seems to imply an unspecified problem exists in `odeint`. I suggest a small rephrase: ``for new code, it is recommended to use `solve_ivp` instead of `odeint`.``

Reply:
To be fair, problems do exist in odeint that probably will never be fixed.

Reply:
They may be if someone cares enough. And they're mostly corner cases as far as I can tell from a quick browse through the issues. `odeint` is very widely used, will not go away, and is still better than `solve_ivp` for certain use cases (see e.g. gh-8257). So I prefer the rephrase.